conda_manager/api/conda_api.py

```diff
@@ -107,10 +108,10 @@ def __init__(self, cmd_list, parse=False, pip=False, callback=None,
         self._timer = QTimer()
         self._process = QProcess()
 
-        self._timer.setInterval(50)
+        self._timer.setInterval(150)
 
         self._timer.timeout.connect(self._communicate)
-        self._process.finished.connect(self._communicate)
+        # self._process.finished.connect(self._communicate)
```

Comment:
If this line is not needed anymore, it should be removed instead of commented out.